src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -37,7 +40,12 @@
           cols="12"
           sm="7"
         >
-          ** Jurisdiction selector goes here **
+          <Jurisdiction
+            :showUsaJurisdictions="true"
+            :initialValue="jurisdictionInitialVal"
+            :errorMessages="jurisdictionErrorMessage"
+            @change="onJurisdictionChange($event)"
+          />
```

Comment:
This component has a lot of whitespace below it (probably the hint/error div) that you should remove.

One way would be to add `hide-details="auto"` to it. Since your use of this component does not display hint or details, you could suck up the bottom with "mt-n4" or something like that.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/4d7b3e46-5541-4795-8c5d-2e1bed81a383)

Reply:
done

Reply:
I think I was wrong here, since you do show an error sometimes:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/ec7aace9-c513-48ad-ad9b-157f3e4a90e8)

The other thing here is that error didn't go away when I selected a jurisdiction...?

Reply:
For the whitespace, I can think of three choices:
1. Use hide-details=auto in the shared component.
2. Leave the whitespace.
3. Use CSS to show the whitespace only if an error is displayed. I did this somewhere a long time ago (before I know about hide-details=auto) so you can maybe find that in Filings UI or Create UI.

Reply:
I am not able to reproduce the "Error not going away after selecting Jurisdiction. I will keep testing....

https://github.com/bcgov/business-create-ui/assets/116035339/587fb9e8-7404-42b4-99c8-a096bd9d2808

Reply:
That's weird. I think that's exactly what I did. I will test again.

Hey, now I found this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/b3b276e5-d475-43e8-86fc-d3a54d37bdd1)

Steps:
1. load https://business-create-dev--pr-684-c6w4g4zv.web.app/continuation-in-business-home?id=TZDikc3p6I
2. go to step 5
3. go to step 1
4. click Undo

Reply:
5. select Alabama
6. notice that the required fields aren't identified

Reply:
Okay. Looking...

Reply:
The issue is `getShowErrors` is set to `true` when Jurisdiction is empty.in some cases  When Jurisdiction is selected, validation message/section invalid was still displayed as `getShowErrors`  = `true` even after Jurisdiction is selected. I have to explicitly set it to `false` after Jurisdiction is selected. See line 375


https://github.com/bcgov/business-create-ui/assets/116035339/1f2e2d2b-3505-4aac-9968-1afd4f3dd70e

Reply:
`showErrors` is a global variable that should be set only when the review page is shown, and never cleared. Don't do this.

It's fine that your component is invalid if the flag has been set, but your component should become valid once data is populated in it.